### PR TITLE
Sign in

### PR DESCRIPTION
**Pull request by khanhhung142**

### CODE REVIEW

File: back-end/src/configs/db.config.ts [hunk 1]
```diff
@@ -18,8 +17,10 @@ export class Database {
         //TO DO
```
Comment (caohaisil201) [inline back-end/src/configs/db.config.ts:17]: TODO: làm gì nhỉ

File: back-end/src/controllers/authentication.controller.ts [hunk 2]
```diff
@@ -57,63 +58,42 @@ export class Auth {
...
-    req: express.Request,
-    res: express.Response
-  ) => {
+  public activateAccount = async (req: Request, res: Response) => {
     try {
       const activateResult = await UsersService.activateAccount(
         req.params.name
       );
       res.json(activateResult);
     } catch (err) {
```
Comment (caohaisil201) [inline back-end/src/controllers/authentication.controller.ts:79]: need to send error

**Review by trivm2472 — APPROVED** [review]
That's seem fine to me

File: front-end/components/SignUpPanel/index.tsx [hunk 3]
```diff
@@ -76,7 +75,7 @@ function SignUpPanel() {
           }
         });
     } catch (error) {
-      console.log(error);
+      // handle error
```
Comment (caohaisil201) [inline front-end/components/SignUpPanel/index.tsx:78]: TODO:

File: front-end/pages/_app.tsx [hunk 4]
```diff
@@ -5,13 +5,12 @@ import { useEffect } from 'react';
 
 function MyApp ({ Component, pageProps }: AppProps) {
   const router = useRouter();
-
   useEffect(() => {
     if (router.pathname === '/_error') {
-      router.push('/404');
-      // router.push('/sign-in');
+      //router.push('/404');
```
Comment (caohaisil201) [inline front-end/pages/_app.tsx:10]: should delete it ?

**Review by caohaisil201 — APPROVED** [review]

**Pull request merged**